app/components/primer/alpha/tooltip.rb

```diff
@@ -8,20 +8,20 @@ module Alpha
     # - `Tooltip` should be rendered through the API of <%= link_to_component(Primer::ButtonComponent)%>, <%= link_to_component(Primer::Beta::Link)%>, or <%= link_to_component(Primer::IconButton)%>. Avoid using `Tooltip` a standalone component unless absolutely necessary (and **only** on interactive elements).
-    # @accessibility
-    #   - Tooltip text must be brief and concise whether it is a label or a description.
-    #   - Tooltip can only hold string content.
-    #   - **Never set tooltips on static, non-interactive elements** like `span` or `div`. Tooltips should only be used on interactive elements like buttons or links to avoid excluding keyboard-only users
-    #   and screen reader users. Use of tooltip through <%= link_to_component(Primer::ButtonComponent) %>, <%= link_to_component(Primer::Beta::Link) %>, or <%= link_to_component(Primer::IconButton) %> will guarantee this.
-    #   - If you must use `Tooltip` as a standalone component, place it adjacent after the trigger element in the DOM. This allows screen reader users to navigate to and copy the tooltip
+    # - Tooltip text must be brief and concise whether it is a label or a description.
+    # - Tooltip can only hold string content.
+    # - Tooltip are not allowed on `disabled` elements due to how `disabled` elements are not keyboard accessible. If you must set a tooltip on a disabled element, use `aria-disabled="true"` and programatically disable the element to ensure the control can be focused.
+    # - **Never set tooltips on static, non-interactive elements** like `span` or `div`. Tooltips should only be used on interactive elements like buttons or links to avoid excluding keyboard-only users
+    # and screen reader users. Use of tooltip through <%= link_to_component(Primer::ButtonComponent) %>, <%= link_to_component(Primer::Beta::Link) %>, or <%= link_to_component(Primer::IconButton) %> will guarantee this.
+    # - If you must use `Tooltip` as a standalone component, place it adjacent after the trigger element in the DOM. This allows screen reader users to navigate to and copy the tooltip
     #   content.
-    #   ### Which type should I set?
-    #   Semantically, a tooltip will either act an accessible name or an accessible description for the element that it is associated with resulting in either a
-    #   `aria-labelledby` or an `aria-describedby` association. The `type` drastically changes semantics and screen reader behavior so follow these guidelines carefully:
-    #   - When there is already a visible label text on the trigger element, the tooltip is likely intended be supplementary, so set `type: :description`.
-    #   The majority of tooltips will fall under this category.
-    #   - When there is no visible text on the trigger element and the tooltip content is appropriate as a label for the element, set `type: :label`.
-    #   `label` type is usually only appropriate for an icon-only control.
+    #
```

Comment:
I noticed that the "Accessibility" section and the "Which type do I set" information is not being rendered on the [doc page for tooltip](https://primer.style/design/components/tooltip/rails). I'm assuming this is due to the ongoing docs migration work, and we're only allowing top-level text to show up.

I think this information is very important to surface so I opted to remove the headings to ensure it shows up in the doc.

Reply:
Hmm that's interesting. I'll look into it. Ideally all this info should make it to the new docsite.

Reply:
Thank you! Happy to add back the headings later!